settings/settings.go

```diff
@@ -1,21 +1,26 @@
 package settings
 
 import (
+	"log"
 	"os"
 	"time"
 
 	"github.com/joho/godotenv"
 )
 
-func InitSettings() {
-	godotenv.Load(".env")
+func InitSettings() error {
+	err := godotenv.Load(".env")
```

Comment:
OPTIONAL: just a small note here.

The purpose of the `.env` file is to populate the process of your program with shell envs easily, just from the file, instead of doing it manually by `export VAR=...` several times. It's good for local development, but for production, I bet you won't use `.env` file.

For example, if your application will be deployed in Kubernetes, you would probably define environment of you pod just by [enumerating it in the manifest file](https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/#define-an-environment-variable-for-a-container) or [mapping](https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/#define-an-environment-variable-for-a-container) pre-created `ConfigMap` to envs.

So, to conclude - I would change logic of getting env variables, for development environment  I would throw an error if .env file is missing, but for prod I would just log that .env file is missing and go on, ignoring error.